eng/pipelines/jobs/windows-build-PR.yml

```diff
@@ -30,14 +30,16 @@ jobs:
           -c $(_BuildConfig)
           /p:TargetArchitecture=${{ parameters.targetArchitecture }}
           /p:SkipTests=${{ parameters.skipTests }}
-      - name: MsbuildSigningArguments
-        value: /p:DotNetSignType=$(SignType)
       - name: TargetArchitecture
         value: ${{ parameters.targetArchitecture }}
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        - name: MsbuildSigningArguments
+          value: /p:DotNetSignType=Test
         - name: _InternalRuntimeDownloadArgs
           value: ''
       - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+        - name: MsbuildSigningArguments
+          value: /p:DotNetSignType=Real
         - group: DotNet-MSRC-Storage
```

Comment:
LGTM. I am pretty sure this can be removed btw.

Reply:
Yep tried a run with it removed and I am not seeing any difference so I've gone ahead and removed this.